content/SCALE/SCALETutorials/SystemSettings/Services/NFSServiceSCALE.md

```diff
@@ -19,39 +19,30 @@ Select **Start Automatically** to activate NFS service when TrueNAS boots.
 
 ![ServicesNFSSettingsScreen](/images/SCALE/22.02/ServicesNFSSettingsScreen.png "Services NFS Options")
 
-### General Options Settings
+### Configuring NFS Service
 
-| Setting | Description |
-|---------|-------------|
-| **Number of servers** | Required. Enter the number of servers to create. Increase if NFS client responses are slow. Keep this less than or equal to the number of CPUs reported by `SYSCTL -n kern.smp.cpus` to limit CPU context switching. |
-| **Bind IP Addresses** | Select IP addresses to listen to for NFS requests. Leave empty for NFS to listen to all available addresses. You must configure static IPs on the interface to appear on the dropdown list. |
+Unless a specific setting is required, we recommend using the default NFS settings.
 
-### NFSv4 Settings
+First enter the number of servers you want to create in **Number of servers**. 
 
-| Setting | Description |
-|---------|-------------|
-| **Enable NFSv4** | Select to switch from NFSv3 to NFSv4. If selected, **NFSv3 ownership model for NFSv4** clears, allowing you to select or leave it clear. |
-| **NFSv3 ownership model for NFSv4** | Becomes selectable after selecting **Enable NFSv4**. Select when NFSv4 ACL support is needed without requiring the client and the server to sync users and groups. |
-| **Require Kerberos for NFSv4** | Select to force NFS shares to fail if the Kerberos ticket is unavailable. |
+Select the IP addressed from the **Bind IP Addresses** dropdown list if you want to use a specific static IP address, or to to list on all available addresses leave this blank.
```

Comment:
..."or to list on all available addresses leave this blank." (Delete duplicate "to")